src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Color {
+    Blue, Green, Dark, Yellow, Grey, Red;
```

Comment:
[code style](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
According to the code style, all constants must be named in upper case. Enum contains constants.